flang/test/Driver/mlink-builtin-bc.f90

```diff
@@ -0,0 +1,18 @@
+
+!----------
+! RUN lines
+!----------
```

Comment:
In the past, folks asked not to add such comments in tests. Let's stick with that.